recommendation_system/ranking_layer/simple.py

```diff
@@ -0,0 +1,66 @@
+from copy import deepcopy
+from typing import List, Dict, Tuple
+
+from recommendation_system.ranking_layer.base import BaseRankingModel
+
+class SimpleRankingModel(BaseRankingModel):
+    def __init__(self) -> None:
+        super().__init__()
+
+    def rank(self, user_features: Dict, candidates: List[List[Dict]], top_k: int) -> List[Dict]:
+        candidates = self.process_data(candidates=candidates)
+        rankings = self.calc_candidate_score(candidates=candidates)
+
+        new_candidates = []
+        for item, score in rankings:
+            new_item = self.get_item(item)
+            new_item['score'] = score
```

Comment:
if you'd calculated score in `calc_candiate_score` func, then maybe you can return item along with `score` field right? thoughts?